src/utilities/general.ts

```diff
@@ -54,6 +55,23 @@ class Util {
 			addRelative ? ` (<t:${timestamp}:R>)` : ""
 		}`
 	}
+
+	/**
+	 * Replies to an interaction or edits the reply if the interaction has already been replied to. Sets ephemeral to undefined if the interaction has already been replied to.
+	 * @param {RepliableInteraction} interaction The interaction to reply to.
+	 * @param {InteractionReplyOptions} content The content to reply with.
+	 */
+	public static replyOrEdit(
+		interaction: RepliableInteraction,
+		content: InteractionReplyOptions
+	) {
+		if (interaction.replied) {
```

Comment:
Ternary statement?

Reply:
No, because you can't do `content.ephemeral = undefined` in a clean way in a ternary.